captcha.php

```diff
@@ -13,11 +13,15 @@
 // Register captcha validator
 v::$validators['captcha'] = function($value, $namespace = NULL) {
   $securimage = new Securimage();
-  if (NULL !== $namespace) {
+  
+  /* if (NULL !== $namespace) {
     $securimage->setNamespace($namespace);
-  }
-
-  return $securimage->check($value);
+  }*/
+    if ($securimage->check($value) == true) {
+      return true;
+    } else {
+      return false;
+    }
 };
 
```

Comment:
Last part is indeed the same.
But I think that this:
if (NULL !== $namespace) {
     $securimage->setNamespace($namespace);
}

is commented out. That caused the problem.